frontend/src/scenes/insights/insightSceneLogic.tsx

```diff
@@ -89,6 +94,7 @@ export const insightSceneLogic = kea<insightSceneLogicType>([
             if (logicInsightId !== insightId) {
                 const oldCache = values.insightCache // free old logic after mounting new one
                 if (insightId) {
+                    // when this line runs we get `Error: [KEA] Logic "scenes.insights.insightLogic.new" can not connect to undefined to request action "setActiveView"`
```

Comment:
I guess that I'd need to do more work to avoid circular dependencies here but thought I'd check if I'm missing some dark magic